test/unit/spec/OktaSignIn_spec.js

```diff
@@ -835,11 +857,192 @@ Expect.describe('OktaSignIn v2 bootstrap', function () {
       });
     });
 
-  });
+    itp('Saves the interaction handle', () => {
```

Comment:
can those cases be covered via testcafe ?

Reply:
these unit tests could actually be moved to jest. I am testing the specific cases where meta is saved or loaded by mocking/spying on the authClient's transactionManager.  There is one case for saving, and several for clearing. (I suspect most of the bugs we will see with this feature will involve cases where we either clear or do not clear inappropriately, so I wanted very low-level coverage. I plan to add even lower-level unit tests on the individual parts of the login flow to verify everything works as expected at all points: OKTA-372526

There is an argument to me made for more higher-level tests in testcafe. These would run the full interaction code flow and verify the correct UI is displayed.

- interaction code flow with no saved interaction handle
- interaction code flow with saved interaction handle
- social auth flow with redirect
- error cases (session expired, invalid interaction handle)

I could add some of these tests to this PR or do a follow-up. What do you think?

Reply:
follow up is fine.
I just think if testcafe can have sufficient test coverage, we don't need those unit test.